client/src/index.ts

```diff
@@ -231,28 +291,68 @@ export class Query<T, R = T> {
     return q;
   }
 
-  // TODO: pagination. Maybe implement different methods for different type of paginations
-  // and one to simply get the first records returned by the query with no pagination.
+  async getPaginated(options?: BulkQueryOptions<T>): Promise<Page<T, R>> {
```

Comment:
I think this is only for private use, right?

Could we make it private and/or put an underscore as prefix? WDYT

Reply:
Since ``Page`` is an entity that we are using publishing on ``getNextPage``/etc... I don't mind exposing it in case someone wants to build their own pagination.